### PR TITLE
minutes meeting 27 April 2016

### PR DESCRIPTION
**Pull request by annefou**
Hi,

Please find the minutes from our last meeting. 

I left empty one item "Post-workshop evaluation form for non-SWC/DC workshops":  I did not have anything meaningful in my notes...

Anne.

### CODE REVIEW

File: meetings/160427_organisers.md [hunk 1]
```diff
@@ -11,4 +11,139 @@
...
+
+- Issues with matplotlib on the Python lesson: Halfdan mentioned in an email that these issues should be reported by Arvind or Hugues as he was not present when it happened. **This issue is still pending**.
+
+- Lex contacted the Research Bazaar: he is now included in their mailing list. **Action closed**.
+
+- We still did not recieve the pre and post-workshop feedbacks for the spatio-temporal workshop (they were both managed by NEON). **Action is pending** and Anne to ask Leah again.
+
+- Involvement of local IT-groups (to install SWC and DC software stack) has been mentioned during the Carpentry meeting with the Library. **The action is still opened** and a list of software will be defined after the next SWC workshop in June.
+
+- 10th Nay one day workshop: Carrie and Christian met and discussed the content of the course. **Action is closed**.
```
Comment (lexnederbragt) [inline meetings/160427_organisers.md]: Nay --> May

File: meetings/160427_organisers.md [hunk 2]
```diff
@@ -11,4 +11,139 @@
...
+##meetups/software therapy/data therapy (‘beyond the workshop’)##
+
+Data therapy subgroup will meet on Friday.  
+
+Some important figures to take into account for the discussions:
+
+- What is the best way to exchange and discuss: discussion list or facebook, twitter, forum, etc.
+
+- we have about 30 instructors (organizers mailing list) that can potentially participate/teach/help to these Data Therapy.
+- We 242 people registered to our discussion mailing list.
```
Comment (lexnederbragt) [inline meetings/160427_organisers.md]: 'We have', or remove 'We'.

File: meetings/160427_organisers.md [hunk 3]
```diff
@@ -11,4 +11,139 @@
...
+- So in total we have about 292 persons (attendees workshop + instructors)
+
+#### Actions ####
+- The summary and minutes of the subgroup meeting will be sent separately.
+
+##AOB##
```
Comment (lexnederbragt) [inline meetings/160427_organisers.md]: AOB? What is that?

**Comment by lexnederbragt** [comment]
"Post-workshop evaluation form for non-SWC/DC workshops" I'm not sure either, but it would be nice if we could have a simple evaluation form...

File: meetings/160427_organisers.md (hunk 3 above)
Comment (annefou) [inline meetings/160427_organisers.md]: AOB means "Any Other Business". I used it here as the discussion John triggered was not in the agenda but still interesting to mention.

**Pull request merged**